openmc_plotter/plotmodel.py

```diff
@@ -116,10 +119,39 @@ def __init__(self):
         # reset random number seed for consistent
         # coloring when reloading a model
         reset_seed()
-
         self.previousViews = []
         self.subsequentViews = []
-        self.defaultView = self.getDefaultView()
+
+        if use_settings_pkl and os.path.isfile('plot_settings.pkl'):
+            with open('plot_settings.pkl', 'rb') as file:
+                model = pickle.load(file)
+
+                # check GUI version
+                if model.version != self.version:
+                    print("WARNING: previous plot settings are for a different "
+                        "version of the GUI. They will be ignored.")
+                    wrn_msg = "Existing version: {}, Current GUI version: {}"
+                    print(wrn_msg.format(model.version, self.version))
+                    view = None
+                else:
+                    view = model.currentView
+
+                    # restore statepoint file
+                    try:
+                        self.statepoint = model.statepoint
+                    except OSError:
+                        msg_box = QMessageBox()
+                        msg = "Could not open statepoint file: \n\n {} \n"
+                        msg_box.setText(msg.format(self.model.statepoint.filename))
+                        msg_box.setIcon(QMessageBox.Warning)
+                        msg_box.setStandardButtons(QMessageBox.Ok)
+                        msg_box.exec_()
+                        self.statepoint = None
+
+                self.defaultView = PlotView(restore_view=view)
+        else:
+            self.defaultView = self.getDefaultView()
+
```

Comment:
This is where I would add something to gather the previous and subsequent views from the `plot_settings.pkl` if they are needed:

```
for view in model.previousViews:
    self.previousViews.append(PlotView(restore_view=view))
```

And the same for `subsequentViews`.

This way these views could also be reset but with correct models.